draft-ietf-webpush-protocol.xml

```diff
@@ -501,14 +499,20 @@ TTL = 1*DIGIT
         </t>
         <t>
           Absence of the TTL header field is interpreted as equivalent to a zero
-          value.  Push messages with a zero TTL indicate that storage is not
-          needed and that the message can be dropped if the user agent isn't
-          immediately available to receive the message.  Push messages with a
-          zero TTL can be delivered very efficiently.
-          <!-- But we need to determine if they can be acknowledged.  Decoupling
-               of delivery and acknowledgment makes this tricky. -->
+          value.  A Push message with a zero TTL is immediately delivered if the
+          user agent is available to receive the message.
         </t>
         <t>
+          The push service is then permitted to immediately remove the push message

```

Comment:
"is then permitted" doesn't really have enough context.  I find that you have to repeat yourself:

> The push service is permitted to immediately remove a push message if it has a zero TTL.